tdrs-backend/localstack-setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Create the bucket used by the Django app
+awslocal s3api create-bucket --bucket $AWS_BUCKET --region $AWS_REGION_NAME
+
+# Enable object versioning on the bucket
+awslocal s3api put-bucket-versioning --bucket $AWS_BUCKET --versioning-configuration Status=Enabled
```

Comment:
This script will create the bucket expected by tests for the Data Files to be uploaded to.

Reply:
Could this be moved to `/scripts`?

Reply:
Yeah I think it could, but `/scripts` right now is all stuff for Circle CI to call - not sure if that was intended or just how it worked out. The main reason I put it here was so it would be at the same level as the backend compose file that calls it.

Reply:
that is just how its worked out so far, the circle CI scripts use to be in the base directory too before I refactored for buildpacks

Reply:
Do we have a preference on location here? While I don't feel strongly, I think the scripts folder would be the best place for it.

Reply:
I think it makes sense to move it into `/scripts` - since it doesn't need to be included in the code packaged up by the buildpacks deployments. I confirmed this still works as expected locally after updating the path in docker compose.